source/draw/DrawState.ooc

```diff
@@ -25,7 +27,6 @@ DrawState: cover {
 	_originReference := FloatPoint2D new(0.0f, 0.0f) // Start of text
 	map: Map = null
 	mesh: Mesh = null
-	opacity := 1.0f
```

Comment:
Obsoleted by alpha filtering since the shader can multiply output alpha with a uniform to preserve source alpha.